### ISSUE TITLE
Desvincular a tabela de dados de n de casos do Brasil da de estados

### ISSUE THREAD

**Issue by piklprado** [covid19br/reports#3]
A tabela de casos confirmados no Brasil vem da totalização da tabela de estados, correto? Acho que é uma dependência que pode quebrar muito fácil, pq basta um erro ou omissão de dados de algum estado para dar problema. Além disso, o dado oficial para o Brasil, consolidado pelo Ministério, muitas vezes é diferente da somas dos estados, por causa de diferenças no horário de fechamento dos boletins.

Então sugiro ter uma tabela para o Brasil independente.

**Comment by gburin** [comment]
Concordo. Essa tabela será puxada de algum lugar ou ela entraria no caso de usarmos uma tabela independente?

**Comment by piklprado** [comment]
Por enquanto acho que fica mais geral se o nome da tabela for um argumento da função. Se/quando a gente estabilizar a fonte da tabela de dados Brasil aí a gente troca por um endereço destas tabela. Ou alternativa é manter uma tabela "brasil.csv" no diretóriod e dados e usar ela quando o argumento for para fazer o relatório do Brasil
